static/app/views/replays/replays.tsx

```diff
@@ -85,22 +89,24 @@ class Replays extends React.Component<Props> {
               </div>
             </HeaderTitle>
           </PageHeader>
-          <Panel>
-            <PanelBody>
-              <DiscoverQuery
-                eventView={this.getEventView()}
-                location={this.props.location}
-                orgSlug={organization.slug}
-              >
-                {data => {
-                  if (data.isLoading) {
-                    return <LoadingIndicator />;
-                  }
-                  return this.renderTable(data.tableData.data);
-                }}
-              </DiscoverQuery>
-            </PanelBody>
-          </Panel>
+
+          <DiscoverQuery
+            eventView={this.getEventView()}
+            location={this.props.location}
+            orgSlug={organization.slug}
+          >
+            {data => {
+              return (
+                <PanelTable
+                  isLoading={data.isLoading}
+                  isEmpty={data.tableData?.length === 0}
```

Comment:
The rows of the table would be `data.tableData?.data`, might want to check that? I have a PR up to fix the types for `data` since I noticed they had become `any`.

Reply:
Oops